src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -261,18 +262,17 @@ private void checkPersonName(Name name, String personType, Appellant appellant)
 
     private void checkPersonAddressAndDob(Address address, Identity identity, String personType, Map<String, Object> ocrCaseData, Map<String, Object> caseData, Appellant appellant) {
 
-        boolean isAddressLine4Present = findBooleanExists(getField(ocrCaseData, personType + "_address_line4"));
+        boolean isAddressLine4Present = findBooleanExists(getField(ocrCaseData, personType + ADDRESS_LINE4));
 
         if (!doesAddressLine1Exist(address)) {
             warnings.add(getMessageByCallbackType(callbackType, personType, getWarningMessageName(personType, appellant) + ADDRESS_LINE1, IS_EMPTY));
         }
         if (!doesAddressTownExist(address)) {
             String addressLine = (isAddressLine4Present) ? ADDRESS_LINE3 : ADDRESS_LINE2;
             warnings.add(getMessageByCallbackType(callbackType, personType, getWarningMessageName(personType, appellant) + addressLine, IS_EMPTY));
-
         }
         if (!doesAddressCountyExist(address)) {
-            String addressLine = (isAddressLine4Present) ? ADDRESS_LINE4 : "_ADDRESS_LINE3_COUNTY";
+            String addressLine = (isAddressLine4Present) ? ADDRESS_LINE4 : LOCAL_ADDRESS_LINE_3_COUNTY;
```

Comment:
Don't really understand this line change. Can you explain what it is? Or is there a test needed to for this change?

Reply:
This is existing constant string I just changed to local constant variable. Its not require though